camelot/view/controls/delegates/colordelegate.py

```diff
@@ -47,4 +47,7 @@ def get_standard_item(cls, locale, model_context):
         item = super(ColorDelegate, cls).get_standard_item(locale, model_context)
         color = editors.ColorEditor.to_qcolor(model_context.value, Qt.GlobalColor.transparent)
         item.setData(color, Qt.ItemDataRole.BackgroundRole)
+        if model_context.value is not None:
+            if isinstance(model_context.value, QtGui.QColor):
```

Comment:
zouden we de colors ook nt in de constant naming context afhandelen ?

dat maakt conversie naar QColor aan de UI side ook straight forward ?

Reply:
Ja, daar was ikmook al aan het denken...